Makefile.tt

```diff
@@ -0,0 +1,57 @@
+include .env
+
+.PHONY: db/migrate db/rollback db/wait-for-postgres db/seed dev env/setup env/teardown test test/docker codebase codebase/fix
+
+db/migrate:
+	make db/wait-for-postgres
+	rails db:migrate
+
+db/rollback:
+	make db/wait-for-postgres
+	rails db:rollback
+
+db/wait-for-postgres:
+	$(shell DATABASE_URL=$(DATABASE_URL) ./bin/wait-for-postgres.sh)
+
+db/seed:
+	rails db:seed
+
+dev:
+	make install-dependencies
+	make env/setup
+	./bin/dev
+
+env/setup:
+	./bin/envsetup.sh
+	rails db:prepare
+
+env/teardown:  # this command will delete data
+	./bin/envteardown.sh
+
+generate-sprite:
+	./bin/svg-sprite
```

Comment:
```suggestion
```

I think fresh apps do not come with svg-sprite GEM 😇

Reply:
https://github.com/nimblehq/rails-templates/pull/359/commits/c4aeed2b779d192b298409e399e41a0d61de34d3 ✅